packages/react-instantsearch-core/src/connectors/__tests__/connectRefinementList.js

```diff
@@ -410,8 +443,14 @@ describe('connectRefinementList', () => {
     });
 
     it('calling searchForItems return the right searchForItems parameters with showMoreLimit', () => {
-      const parameters = searchForFacetValues(
-        { attribute: 'ok', limit: 15, showMoreLimit: 25, showMore: true },
+      const parameters = connect.searchForFacetValues(
+        {
+          attribute: 'ok',
+          limit: 15,
+          showMoreLimit: 25,
+          showMore: true,
+          contextValue,
```

Comment:
It's useless in that case, we don't use the context value inside the `searchForFacetValues`.